scripts/scoring_test.py

```diff
@@ -0,0 +1,29 @@
+import requests
+from bs4 import BeautifulSoup
+import numpy as np
+
+DEBUG_URL = 'http://127.0.0.1:5000'
+
+def get_word_score(language1, language2, spelling):
+    response = requests.post(f'{DEBUG_URL}/translitteration/', data = {'language1': language1, 'language2': language2, 'spelling': spelling})
+    data = response.text
+    soup = BeautifulSoup(data, "html5lib")
```

Comment:
This is a weird logic in my opinion. You are getting from your backend a webpage that you are then parsing by hand. It would be much better to either do an http request to your backend returning a JSON directly or use the class that compute this harmonic mean. Let me know if it's not clear.